rxandroidble/src/main/java/com/polidea/rxandroidble2/RxBlePhy.java

```diff
@@ -0,0 +1,76 @@
+package com.polidea.rxandroidble2;
+
+import androidx.annotation.NonNull;
+
+import com.polidea.rxandroidble2.internal.RxBleLog;
+
+import java.util.EnumSet;
+import java.util.Iterator;
+
+public enum RxBlePhy {
+    /**
+     * Unknown Bluetooth PHY. Used if the PHY couldn't be determined.
+     *
+     * @implNote If this value is used when calling the setPreferredPhy then it will default to PHY_1M
+     */
+    PHY_UNKNOWN(0),
+
+    /**
+     * Bluetooth LE 1M PHY.
+     */
+    PHY_1M(1),
+
+    /**
+     * Bluetooth LE 2M PHY.
+     */
+    PHY_2M(1 << 1),
+
+    /**
+     * Bluetooth LE Coded PHY.
+     */
+    PHY_CODED(1 << 2);
+
+    private final int value;
+
+    RxBlePhy(final int value) {
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    @NonNull
+    public static RxBlePhy fromInt(final int i) {
+        for (final RxBlePhy entry : RxBlePhy.values()) {
+            if (entry.getValue() == i) {
+                return entry;
+            }
+        }
+        RxBleLog.w("%d is not a valid PHY value.", i);
+        return RxBlePhy.PHY_UNKNOWN;
+    }
+
+    public static int enumSetToInt(EnumSet<RxBlePhy> set) {
```

Comment:
This is a bit vague imo. Maybe `enumSetToValuesMask`?
Additionally, since this is a public function, maybe it is worth to have a javadoc.